src/main/java/org/kohsuke/github/GHTeam.java

```diff
@@ -166,12 +179,18 @@ public GHDiscussion getDiscussion(long discussionNumber) throws IOException {
     /**
      * Retrieves the current members.
      *
+     * @param role
+     *            the role to filter the result by
+     *
      * @return the paged iterable
      * @throws IOException
      *             the io exception
      */
-    public PagedIterable<GHUser> listMembers() throws IOException {
-        return root.createRequest().withUrlPath(api("/members")).toIterable(GHUser[].class, item -> item.wrapUp(root));
```

Comment:
This is an API change. Please retain the existing public method.

Reply:
I have readded the original public method.